packages/kolibri/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "kolibri",
+  "version": "0.12.0-dev.1",
+  "description": "The Kolibri core API",
+  "repository": "github.com/learningequality/kolibri",
+  "author": "Learning Equality",
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "build": "node ./build/index.js"
+  },
+  "dependencies": {
+    "clipboard": "^2.0.1",
+    "css-element-queries": "1.0.1",
+    "filesize": "^3.3.0 ",
+    "fontfaceobserver": "^2.0.13",
+    "frame-throttle": "^2.0.1",
+    "js-cookie": "^2.1.2",
+    "keen-ui": "https://github.com/learningequality/Keen-UI/",
+    "lockr": "0.8.4",
+    "lodash": "^4.17.4",
+    "loglevel": "^1.4.1",
+    "purecss": "^0.6.2",
+    "rest": "^1.3.2",
+    "screenfull": "^3.3.2",
+    "seededshuffle": "^0.1.1",
+    "shave": "^2.1.3",
+    "vue": "^2.5.17",
+    "vue-router": "^2.1.1",
+    "vuex": "^3.0.1"
+  },
+  "devDependencies": {
+    "kolibri-tools": "0.12.0-dev.1"
```

Comment:
this is fine because it's an internal dependency, but if we ever make it public I'd prefer `kolibri-dev-tools`